third_party/move/move-model/bytecode/src/stackless_bytecode.rs

```diff
@@ -475,15 +475,13 @@ impl Bytecode {
             Bytecode::Ret(..)
                 | Bytecode::Jump(..)
                 | Bytecode::Abort(..)
+                | Bytecode::Branch(..)
                 | Bytecode::Call(_, _, Operation::Stop, _, _)
         )
     }
 
     pub fn is_conditional_branch(&self) -> bool {
-        matches!(
-            self,
-            Bytecode::Branch(..) | Bytecode::Call(_, _, _, _, Some(_))
-        )
+        matches!(self, Bytecode::Call(_, _, _, _, Some(_)))
```

Comment:
Same comment as above: why is `Branch` not a conditional branch?

Reply:
Done